TestFoundation/TestNSLocale.swift

```diff
@@ -110,10 +110,29 @@ class TestNSLocale : XCTestCase {
         XCTAssertTrue(locale == localeCopy)
     }
 
-     func test_availableIdentifiers() {
-        XCTAssertNoThrow(Locale.availableIdentifiers)
+    func test_staticProperties() {
+        let euroCurrencyCode = "EUR"
+        let spainRegionCode = "ES"
+        let galicianLanguageCode = "gl"
+        let galicianLocaleIdentifier = Locale.identifier(fromComponents: [NSLocale.Key.languageCode.rawValue: galicianLanguageCode,
```

Comment:
Why the hoop jump through `languageCode.rawValue` here? The argument type is `[String:String]`

Reply:
Any suggestions? 🤔

Reply:
It looks like the types are all consistent with Darwin :( so, not an issue that concerns this patch specifically.